.github/workflows/codex-implement-issue.yml

```
@@ -0,0 +1,191 @@
+            --json number,title,body,author,labels,url \
+            > tmp/codex-issue-context.md
+
+      - name: Implement with Codex
+        id: codex
+        uses: openai/codex-action@v1
+        with:
+          openai-api-key: ${{ secrets.OPENAI_API_KEY }}
+          model: gpt-5.6-sol
+          effort: high
+          prompt-file: .github/codex/implement-issue.md
+          permission-profile: read-write
```

actionlint (1.7.12)

[error] input "permission-profile" is not defined in action "openai/codex-action@v1". available inputs are "allow-bots", "allow-users", "codex-args", "codex-home", "codex-user", "codex-version", "effort", "model", "openai-api-key", "output-file", "output-schema", "output-schema-file", "prompt", "prompt-file", "responses-api-endpoint", "safety-strategy", "sandbox", "working-directory"

(action)